main.py

```diff
@@ -50,45 +50,30 @@ def main():
         # Set the input_file variable to the 2nd argument (input file path).
         input_file = sys.argv[1]
 
+        if not (os.path.isfile(input_file)):
+            print("ERROR: Path not valid.")
+
         # If the path is a legitimate file...
-        if (os.path.isfile(input_file)):
-            # Loop until break is called.
-            while True:
-                # Obtain user input.
-                to_trim = input("Do you want to trim this file? (y/n)\n")
-                if not (to_trim == "y" or to_trim == "n"):
-                    print("Invalid input, please enter either 'y' or 'n'.")
-                else:
-                    break
+        else:
+            to_trim = set_trim_status()
+            to_watermark = set_watermark_status()
```

Comment:
I think it's cleaner to work with these as Booleans rather than `"y"`/`"n"` strings